### ISSUE TITLE
xRemoteFile verbose output does not display progress correctly

### ISSUE THREAD

**Issue by mgreenegit** [PowerShell/xPSDesiredStateConfiguration#165]
When xRemoteFile is in the process of downloading, the verbose output displays bytes transferred but does not display the progress bar or time remaining.

Example output:

```
Writing web request
    Writing request stream... (Number of bytes written: 251558324)
    [
    00:00:00 remaining.

    [node] Set-TargetResource
```

**Comment by TravisEz13** [comment]
This is an issue with LCM.  This progress is reported by a built-in CmdLet, Invoke-WebRequest.  This would be better reported in [PowerShell User Voice](https://windowsserver.uservoice.com/forums/301869-powershell)

**Comment by maxxwizard** [comment]
The progress bar is really broken: the "Number of bytes written" value keeps increasing forever.

I upvoted 2 issues that seem similar to this issue's behavior:
- https://windowsserver.uservoice.com/forums/301869-powershell/suggestions/11087604-add-top-level-progress-information-to-invoke-comma
- https://windowsserver.uservoice.com/forums/301869-powershell/suggestions/11088438-expand-windowsimage-fails-to-show-progress

Guess PowerShell team just needs to fix LCM's reporting.

**Comment by johlju** [comment]
This is being resolved in PR #390.
